.dockerignore

```
@@ -4,3 +4,6 @@
 node_modules
```

Just realized we probably shouldn't copy over `dist` during the build step-- in general `.dockerignore` should hew as close to `.gitignore` as possible to keep it consistent across environments.
```suggestion
node_modules
dist
```